content/python/tags.py.md

```diff
@@ -0,0 +1,215 @@
+---
+title: "Working with Tags"
+description: "A few examples on interacting with Tags"
+
+date: "2020-02-19"
+classes: 
+    - "SoftLayer_Tags"    
+tags:
+    - "Tags"
+---
+
+[Tags](https://https://sldn.softlayer.com/reference/services/SoftLayer_Tags/)
+
+### Get all tags
+
+```python
+import SoftLayer
+import json
+
+client = SoftLayer.create_client_from_env()
+account_service = client['Account']
+
+try:
+    response = account_service.getTags()
+    print(json.dumps(response, sort_keys=True, indent=2, separators=(',', ': ')))
+except SoftLayer.SoftLayerAPIError as e:
+    print("Unable to list the response for the package: \nfaultCode= %s, \n \nfaultString= %s"
+          % (e.faultCode, e.faultString))
+```
+
+### Autocomplete tag inputted by a user.
```

Comment:
While ["inputted"](https://www.merriam-webster.com/words-at-play/is-inputted-a-word) is apparently a real world and what is written in the autoComplete documentation, I don't like it.  

This method also doesn't really autoComplete, although I'm sure that is what the results are used for.

Change the title here to  "### Simple Tag Search" since that is what this function really does.